numba_cuda/numba/cuda/decorators.py

```diff
@@ -44,7 +44,10 @@ def jit(func_or_sig=None, device=False, inline=False, link=[], debug=None,
        Useful for increasing occupancy.
     :param opt: Whether to compile from LLVM IR to PTX with optimization
                 enabled. When ``True``, ``-opt=3`` is passed to NVVM. When
-                ``False``, ``-opt=0`` is passed to NVVM. Defaults to ``True``.
+                ``False``, ``-opt=0`` is passed to NVVM. Defaults to ``None``.
+                The default value will be reset by environment variable
+                ``NUMBA_OPT``. If ``0``, ``opt`` is reset as ``False``;
+                otherwise, it is reset to ``True``.
```

Comment:
I think the original documentation needlessly goes into implementation details - users shouldn't even have to know what NVVM is, necessarily. I think it would be simpler to understand as:

> Whether to compile with optimization enabled. If unspecified, the ``OPT`` configuration variable is consulted; all non-zero values will enable optimization.

Reply:
simplified the doc. also added ``NUMBAOPT`` in it.